src/components/Articles.jsx

```diff
@@ -4,19 +4,31 @@ import ArticleCard from './ArticleCard'
 
 const Articles = () => {    
     const [articles, setArticles] = useState([])
-    const [isLoading, setIsLoading] = useState(true);
+    const [isLoading, setIsLoading] = useState(false);
+    const [isError, setIsError] = useState(false);
 
     useEffect(() => {
+        setIsLoading(true);
+        setIsError(false);
+
         getArticles()
         .then((articles) => {
-            setIsLoading(false)
-          setArticles(articles)
-
+            setIsLoading(false);
+            setIsError(false);
+            setArticles(articles)
+        })
+        .catch((error) => {
+            setIsLoading(false);
+            setIsError(true);
+            console.log('Load articles error: ', error)
         })
     }, [])
 
     if (isLoading)
-        return <p>loading...</p>
+        return <article className="articles-cards"><p>Loading articles...</p></article>
+
+    if (isError)
+        return <article className="articles-cards"><p>Error while loading all articles...</p></article>
 
     return (
         <article className="articles-cards">
```

Comment:
Please make sure the img tag has an "alt" attribute for accessibility🙂

Reply:
done